src/utils.test.js

```diff
@@ -120,7 +120,22 @@ const getPrettierOptionsFromESLintRulesTests = [
   },
   {rules: {'max-len': 2}, options: {printWidth: 80}},
   {rules: {'comma-dangle': [2, 'never']}, options: {trailingComma: 'none'}},
-  {rules: {'comma-dangle': [2, 'always']}, options: {trailingComma: 'all'}},
+  {rules: {'comma-dangle': [2, 'always']}, options: {trailingComma: 'es5'}},
```

Comment:
Changing the test this way means this would be a breaking change and I would prefer to avoid that for this feature. Could you make it so that the current tests pass but your use case is still supported?